addons/addon-base-raas-ui/packages/base-raas-ui/src/models/studies/Study.js

```diff
@@ -56,7 +92,7 @@ const Study = types
 
     getPermissionsStore() {
       if (!self.permissionsStore) {
-        self.permissionsStore = StudyPermissionsStore.create({ studyId: self.id });
+        self.permissionsStore = StudyPermissionsStore.create({ studyId: self.id, studyAccessType: self.accessType });
```

Comment:
Is it possible not to introduce the property 'studyAccessType' at the permissions entity level?  can you make it a derived property, instead?